docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -9,9 +13,9 @@ title: Daemon Sets
 
 ## What is a DaemonSet?
 
-A _DaemonSet_ ensures that all (or some) nodes run a copy of a pod.  As nodes are added to the
-cluster, pods are added to them.  As nodes are removed from the cluster, those pods are garbage
-collected.  Deleting a DaemonSet will clean up the pods it created.
+A _DaemonSet_ ensures that all (or some) Nodes run a copy of a Pod.  As nodes are added to the
+cluster, Pods are added to them.  As nodes are removed from the cluster, those Pods are garbage
+collected.  Deleting a DaemonSet will clean up the Pods it created.
```

Comment:
Maybe we should mention that this documentation is for the behavior introduced in `apps/v1beta2`, in case people try to use an older api version and expect this behavior.

Reply:
this is GC. It's been in since 1.6 and doesn't have anything to do with the new API